### ISSUE TITLE
File Not Found

### ISSUE THREAD

**Issue by RahulKalsangra** [RathanakSreang/sudoku_solver#1]
In your Train.py you use suduko_an.png for testing purpose but the images/sudoku_an is not found in that directory .
plz check it and upload that image in image directory.

**Comment by NischayaSharma-iamin** [comment]
Yeah, rename the file in train,py from ```images/suduko_an.png``` to ```images/suduko.png```

**Comment by RahulKalsangra** [comment]
> Yeah, rename the file in train,py from `images/suduko_an.png` to `images/suduko.png`

but after that also it not works there are lots of things to change...

**Comment by NischayaSharma-iamin** [comment]
> 
> 
> > Yeah, rename the file in train,py from `images/suduko_an.png` to `images/suduko.png`
> 
> but after that also it not works there are lots of things to change...

Train worked for me after that little change.

**Comment by RahulKalsangra** [comment]
Yeah , training part will be working fine
